presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -43,6 +43,35 @@ General properties
  * **Description:** Buffer size for IO writes while collecting pipeline results from cluster node. Increasing this value may improve the speed of IO operations, but will take memory away from other functions. Buffered data will be lost if the node crashes, so using a large value is not recommended when the environment is unstable.
 
 
+.. _tuning-spilling:
+
+Properties controlling spilling
+-------------------------------
+
+``experimental.spill-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Boolean``
+ * **Default value:** ``false``
+ * **Description:** Enable :ref:`spilling <spilling-toggle>`. This config property can be overridden by ``spill_enabled`` session property.
+
+
+``experimental.spiller-spill-path``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``String``
+ * **Default value:** ``java.io.tmpdir/presto/spills``
+ * **Description:** Directory where spilled content will be written.
+
+
+``experimental.spiller-threads``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ * **Type:** ``Integer``
+ * **Default value:** ``4``
+ * **Description:** Number of spiller threads. Increase this value if the default one is not able to saturate underlying spilling device (for example when using raid matrix with multiple disks).
```

Comment:
"Increase this value if the default is not able to saturate the underlying spilling device (for example, when using a RAID matrix with multiple disks)"
btw, is there any additional configuration/caveats necessary for JBOD?